debian/bionic/foreman/control

```diff
@@ -19,8 +19,8 @@ Depends: ruby, ruby2.5, bundler (>= 1.3), zlib1g-dev,
          rake (>=0.8.4), ${misc:Depends}
 Recommends: foreman-proxy, foreman-debug (= ${binary:Version}), python-minimal
 Conflicts: ruby-activerecord-deprecated-finders, ruby-foreman-tasks (<< 0.10.1), ruby-sequel-pg
-Replaces: foreman-compute
-Breaks: foreman-compute
+Replaces: foreman-compute, foreman-mysql2
```

Comment:
I don't think this is a good idea. If the user hasn't migrated, this like breaks things. I'd rather it doesn't work and tell the user to remove `foreman-mysql2` in the migration guide.

Reply:
seems reasonable to me. let's make sure people explicitly remove the package when upgrading

Reply:
updated.